src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('thead');
+
+const tablebody = document.querySelectorAll('tbody >tr');
+const tbody = document.querySelector('tbody');
+
+const thead = document.querySelector('thead>tr');
+
+tableHeader.addEventListener('click', (events) => {
```

Comment:
The plural in this variable name is confusing because it says that the variable contains an array, but there is an object
```suggestion
tableHeader.addEventListener('click', (e) => {
```

event / e / customEvent etc.